mscore/instrwidget.cpp

```diff
@@ -362,6 +362,8 @@ QString InstrumentTemplateListItem::text(int col) const
 InstrumentsWidget::InstrumentsWidget(QWidget* parent)
    : QWidget(parent)
       {
+      storeChangeInGenre = false;
```

Comment:
Why is this needed?
Couldn't it be set directly to true?

Reply:
This variable is needed because when the InstrumentsWidget is being created, the selected instrument genre is changed by some other scripts a couple of times. I want to be able to save changes to the genre, but only ones made by the user. So, when loading is complete and this variable has been set to true, only then will any genre changes made be stored, because they are user-made changes, not automatic ones.

Reply:
The variable isn't needed though if you see my reply below I guess